test/tests/meshgenerators/poly2tri_mesh_generator/tests

```diff
@@ -0,0 +1,88 @@
+[Tests]
+  [poly2tri]
+    design = 'meshgenerators/Poly2TriMeshGenerator.md'
+    issues = '#20192'
+    requirement = "The system shall have the capability of triangulating a polyline boundary"
```

Comment:
this is interesting use of requirements... I think it may not display as you expect

Reply:
How so?  I was trying to copy the way the detail parameter gets used in https://mooseframework.inl.gov/framework/documenting.html#f9d79bc2-a119-4f24-994a-33968aa8f371

Where can I see how it gets displayed?  Is it rendered in https://mooseframework.inl.gov/docs/PRs/21800/site/ anywhere?

Reply:
See https://mooseframework.inl.gov/docs/PRs/21800/site/sqa/framework_rtm.htm
<img width="642" alt="Screen Shot 2022-08-24 at 1 44 10 PM" src="https://user-images.githubusercontent.com/22062133/186509027-889c6b31-6b6b-41aa-b0e4-444e7c6645d6.png">
l

Reply:
That broken paren is certainly unacceptable, and in that format the detail definitely reads better in general if it's considered an extension of the requirement rather than of the previous detail.  I'll fix.